R/asis-nodes.R

```diff
@@ -320,3 +320,57 @@ protect_footnotes <- function(body, ns = md_ns()) {
   copy_xml(body)
 }
 #nocov end
+# CURLY ------------------
+find_curly <- function(body, ns) {
+   i <- ".//md:text[not(@asis) and contains(text(), '{')]"
+   xml2::xml_find_all(body, i, ns = ns)
+}
+
+fix_curly <- function(curly) {
```

Comment:
clearly my regex level is not on par with this file :sweat_smile:

Reply:
I call BS on this comment! I have never known how to work with the regmatches and gregexpr functions and I commend your ability to do this!

Reply:
I can't even remember how I got it to work tbh :joy_cat: